src/Maestro/Maestro.Web/Controllers/AzDevController.cs

```diff
@@ -46,5 +46,20 @@ public async Task<IActionResult> GetBuildStatus(string account, string project,
                         Convert.ToBase64String(Encoding.UTF8.GetBytes(":" + token)));
                 });
         }
+
+        [HttpGet("getFile/{account}/{project}/{repo}")]
+        public async Task<IActionResult> GetVersionDetailsFile(string account, string project, string repo)
```

Comment:
Isn't the version important? This is going to fetch some arbitrary file version that might not have anything to do with whatever someone is looking at. Unless this is a "master only" feature.